bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -59,17 +59,12 @@ You will get the most out of this lesson if you follow along with the examples a
 **Open a bash shell.**
 If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.
 
-<div class = "important">
-<b style="color: rgb(var(--color-highlight));">Important note</b><br>
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
 
-We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.
+It can be scary to interact with files directly from the command line. To ensure that we aren't touching any of the important files on your computer, you will be downloading a small directory to experiment with. 
```

Comment:
That being said, I'm not convinced this entire thing should be in the care box. Should perhaps just the encouragement/emotional care parts be in the box (so the first sentence, with maybe some sort of follow up/more general reassurance)? And then "To ensure .... to start over if you want" in the standard body of the module since that's more general information about how we're approaching this lesson? 

I could imagine some learners who maybe don't have as much imposter syndrome or anxiety about these things skimming over the care boxes -- the second half of what's in there currently is still information I think it's important for them to have regardless of if they care for the more emotionally-focused half

Reply:
Thank you for (as always) giving such thoughtful feedback!

I've updated it to incorporate these comments, separating out the care box from the more general information.